ui/app/AppLayouts/Chat/controls/community/EnsPanel.qml

```diff
@@ -55,12 +55,23 @@ ColumnLayout {
                         && !alreadyUsed
             }
         }
+        onVisibleChanged: {
+            if(visible)
+                forceActiveFocus()
+        }
+        onKeyPressed: {
```

Comment:
Better use [TextInput::onAccepted](https://doc.qt.io/qt-5/qml-qtquick-textinput.html#accepted-signal)

Reply:
Same here, since the underlying text component is `TextEdit`